File: 8 kyu/Array plus array/solution.js
function arrayPlusArray(arr1, arr2) {
    // 💬 Concatenate our 2 arrays and reduce it to get the sum
    return [...arr1, ...arr2].reduce((acc, cur) => (acc += cur));
}

const result = arrayPlusArray([1, 2, 3], [4, 5, 6]); // 🥚
console.log(result); // 🐣 21

File: 8 kyu/Array plus array/solution.cs
using System.Linq;

public static class Kata
{
  public static int ArrayPlusArray(int[] arr1, int[] arr2)
  {
    var myArrays = arr1.Concat(arr2).ToArray();
    var sum = 0;
    for(int i=0; i< myArrays.Length; i++ ){
      sum += myArrays[i];
    }
    return sum;
  }
}
